src/Paket.Core/Cultures.fs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Paket
+open System
+open System.Collections.Generic
+open System.Globalization
+
+[<CompilationRepresentation (CompilationRepresentationFlags.ModuleSuffix)>]
```

Comment:
This is needed for Cache module because there is also a Cache type under same namespace.

F# will coallesce both to expose them as Cache but the module need to be given that attribute which under the hood suffixes the static class with Module.

We don't need it for this Cultures since there is no Cultures type that would conflict.